fhir-path/src/main/java/com/ibm/fhir/path/function/MemberOfFunction.java

```diff
@@ -150,7 +148,7 @@ public int getMaxArity() {
 
     private boolean contains(Map<String, Set<String>> codeSetMap, Coding coding) {
         String system = (coding.getSystem() != null) ? coding.getSystem().getValue() : null;
-        String version = (coding.getVersion() != null) ? coding.getVersion().getValue() : FHIRRegistry.getInstance().getLatestVersion(system, CodeSystem.class);
+        String version = (coding.getVersion() != null) ? coding.getVersion().getValue() : null;
```

Comment:
is this change related?

Reply:
Only tangentially related. There were warnings associated with a bug in the CARIN BB IG because we were defaulting an unknown version to Latest Version was causing superfluous warnings to be generated. This change makes the code a bit more tolerant.